x/qgb/orchestrator/loader.go

```diff
@@ -0,0 +1,56 @@
+package orchestrator
+
+import "github.com/celestiaorg/celestia-app/x/qgb/types"
+
+type LoaderI interface {
+	Start() error
+	Stop() error
+	GetDataCommitmentConfirms(nonce uint64) ([]types.MsgDataCommitmentConfirm, error)
+	GetDataCommitmentConfirmByOrchestratorAddress(nonce uint64, orch string) (types.MsgDataCommitmentConfirm, error)
+	GetDataCommitmentConfirmByEthereumAddress(nonce uint64, ethAddr string) (types.MsgDataCommitmentConfirm, error)
+	GetValsetConfirms(nonce uint64) ([]types.MsgValsetConfirm, error)
+	GetValsetConfirmByEthereumAddress(nonce uint64, ethAddr string) (types.MsgValsetConfirm, error)
+	GetValsetConfirmByOrchestratorAddress(nonce uint64, orch string) (types.MsgValsetConfirm, error)
+}
+
+type InMemoryLoader struct {
+	store ConfirmStore
+}
```

Comment:
[question] this wrapper just appears to change the name of the confirmstore, is that its purpose? is this something we have plans to iterate on? either is 100% fine with me, just want to be explicit :slightly_smiling_face:

Reply:
yes, there will be at least 2 implementations of that interface. Also, the name will be updated to  match exactly what's needed. 
This PR only introduces somehow the general structure